locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
-    "Saint Pierre And Miquelon": "St. Pierre and Miquelon",
-    "Saint Vincent And Grenadines": "St. Vincent and Grenadines",
+    "Romania": "Romanya",
+    "Run Action": "Eylem Çalıştır",
+    "Russian Federation": "Rusya Federasyonu",
+    "Rwanda": "Ruanda",
+    "Saint Barthelemy": "Aziz Barthelemy",
+    "Saint Helena": "Aziz Helena",
+    "Saint Kitts And Nevis": "St. Kitts ve Nevis",
+    "Saint Lucia": "Aziz Lucia",
+    "Saint Martin": "Aziz Martin.",
+    "Saint Pierre And Miquelon": "St. Pierre ve Miquelon",
+    "Saint Vincent And Grenadines": "St. Vincent ve Grenadinler",
     "Samoa": "Samoa",
     "San Marino": "San Marino",
-    "Sao Tome And Principe": "São Tomé and Príncipe",
-    "Saudi Arabia": "Saudi Arabia",
+    "Sao Tome And Principe": "São Tomé ve Principe",
+    "Saudi Arabia": "Suudi Arabistan",
     "Save": "Kaydet",
     "Saved.": "Kaydedildi.",
-    "Search": "Search",
-    "Select A New Photo": "Select A New Photo",
-    "Select Action": "Select Action",
-    "Select All": "Select All",
-    "Select All Matching": "Select All Matching",
+    "Search": "Aramak",
+    "Select A New Photo": "Yeni Bir Fotoğraf Seçin",
+    "Select Action": "Eylem Seç",
+    "Select All": "Tümünü Seç",
+    "Select All Matching": "Tüm Eşleşmeleri Seç",
     "Send Password Reset Link": "Parola Sıfırlama Bağlantısı Gönder",
     "Senegal": "Senegal",
-    "September": "September",
-    "Serbia": "Serbia",
+    "September": "Eylül",
+    "Serbia": "Sırbistan",
     "Server Error": "Sunucu Hatası",
     "Service Unavailable": "Hizmet Kullanılamıyor",
-    "Seychelles": "Seychelles",
-    "Show All Fields": "Show All Fields",
-    "Show Content": "Show Content",
-    "Show Recovery Codes": "Show Recovery Codes",
-    "Showing": "Showing",
+    "Seychelles": "Şeysel",
+    "Show All Fields": "Tüm Alanları Göster",
+    "Show Content": "İçeriği Göster",
+    "Show Recovery Codes": "Kurtarma Kodlarını Göster",
+    "Showing": "Gösteri",
     "Sierra Leone": "Sierra Leone",
-    "Singapore": "Singapore",
+    "Singapore": "Singapur",
     "Sint Maarten (Dutch part)": "Sint Maarten",
-    "Slovakia": "Slovakia",
-    "Slovenia": "Slovenia",
-    "Solomon Islands": "Solomon Islands",
-    "Somalia": "Somalia",
-    "Something went wrong.": "Something went wrong.",
-    "Sorry! You are not authorized to perform this action.": "Sorry! You are not authorized to perform this action.",
-    "Sorry, your session has expired.": "Sorry, your session has expired.",
-    "South Africa": "South Africa",
-    "South Georgia And Sandwich Isl.": "South Georgia and South Sandwich Islands",
-    "South Sudan": "South Sudan",
-    "Spain": "Spain",
+    "Slovakia": "Slovakya",
+    "Slovenia": "Slovenya",
+    "Solomon Islands": "Solomon Adaları",
+    "Somalia": "Somali",
+    "Something went wrong.": "Bir şeyler yanlış gitti.",
+    "Sorry! You are not authorized to perform this action.": "Üzgünüm! Bu eylemi gerçekleştirme yetkiniz yoktur.",
```

Comment:
`"Sorry! You are not authorized to perform this action.": "Üzgünüm! Bu eylemi gerçekleştirme yetkiniz yok.",`